batchflow/models/tf/unet.py

```diff
@@ -58,14 +66,7 @@ class UNet(EncoderDecoder):
     def default_config(cls):
         config = super().default_config()
 
-        config['body/encoder/num_stages'] = 4
-        config['body/encoder/order'] = ['block', 'skip', 'downsampling']
-        config['body/encoder/blocks'] += dict(layout='cna cna', kernel_size=3, filters=[64, 128, 256, 512])
-        config['body/embedding'] += dict(layout='cna cna', kernel_size=3, filters=1024)
-        config['body/decoder/order'] = ['upsampling', 'combine', 'block']
-        config['body/decoder/blocks'] += dict(layout='cna cna', kernel_size=3, filters=[512, 256, 128, 64])
-
-        config['loss'] = 'ce'
+        config['body/n_outputs'] = 1
```

Comment:
Why did you delete most of the default config?

Reply:
Thanks, I've edited wrong class.